cmds/init/init.go

```diff
@@ -49,29 +49,34 @@ func main() {
 
 	// In earlier versions we just had src/cmds. Due to the Go rules it seems we need to
 	// embed the URL of the repo everywhere. Yuck.
-	if commands, err := ioutil.ReadDir(path.Join("/src", uroot.CmdsPath)); err == nil {
-		for _, v := range commands {
-			name := v.Name()
-			if name == "installcommand" || name == "init" {
-				continue
-			} else {
-				destPath := path.Join("/buildbin", name)
-				source := "/buildbin/installcommand"
-				if err := os.Symlink(source, destPath); err != nil {
-					log.Printf("Symlink %v -> %v failed; %v", source, destPath, err)
-				}
-				if *ludicrous {
-					log.Printf("Symlink %v -> %v", source, destPath)
-				}
+	c, err := filepath.Glob("/src/github.com/u-root/u-root/cmds/[a-z]*")
+	if err != nil || len(c) == 0 {
+		log.Printf("In a break with tradition, you seem to have NO u-root commands: %v", err)
+	}
+	o, err := filepath.Glob("/src/*/*/*")
```

Comment:
@hugelgupf The changes look good to me. I've tried both on chroot and qemu. The codes works fine with starting up elvish. 

For the /src/*/*/* glob, Ron is willing to accept that for now, as one of the u-root rules.